### PR TITLE
Cleanup files and fix duplicated instance problem

### PR DESCRIPTION
**Pull request by FinalAngel**
WIP

### CODE REVIEW

File: djangocms_googlemap/static/djangocms_googlemap/js/djangocms.googlemaps.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright https://github.com/divio/djangocms-googlemap
+ */
+
+var djangocms = window.djangocms || {};
```
Comment (FinalAngel) [inline djangocms_googlemap/static/djangocms_googlemap/js/djangocms.googlemaps.js]: @vxsx we have to find a generic way of name spacing the addons (also in light of the LTS cleanups) WDYT of this approach?

**Comment by GaretJax** [comment]
LGTM

**Comment by lory87** [comment]
lgtm

**Comment by Brankub** [comment]
👍

**Pull request merged**